### PR TITLE
fix(Sheet): lock focus inside sheet container when content is not interactive

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-2018)

### CODE REVIEW

File: src/sheet-common.tsx [hunk 1]
```diff
@@ -237,6 +237,7 @@ const Sheet = React.forwardRef<HTMLDivElement, SheetProps>(({onClose, children,
                     {...dragableSheetProps}
                     {...getPrefixedDataAttributes(dataAttributes, 'Sheet')}
                     ref={ref}
+                    tabIndex={-1}
```
Comment (atabel) [inline src/sheet-common.tsx:240]: I think the focus should be moved to the element with role=dialog

Comment (marcoskolodny) [reply src/sheet-common.tsx]: You're right. Done :)

File: package.json [hunk 2]
```diff
@@ -79,7 +79,7 @@
         "@swc/cli": "^0.1.62",
         "@swc/core": "^1.3.95",
         "@swc/jest": "^0.2.29",
-        "@telefonica/acceptance-testing": "5.0.0",
+        "@telefonica/acceptance-testing": "5.1.0",
```
Comment (marcoskolodny) [inline package.json:82]: unrelated, just upgrading this lib to have the changes in https://github.com/Telefonica/js-toolbox/pull/107

**Review by pladaria — APPROVED** [review]

**Pull request merged**